ocrd-tool.json

```diff
@@ -4,19 +4,25 @@
   "tools": {
     "ocrd-olena-binarize": {
       "executable": "ocrd-olena-binarize",
-      "description": "OLENA's binarization algos for OCR-D",
+      "description": "OLENA's binarization algos for OCR-D (on page-level)",
```

Comment:
What would have to be done to expose them on non-page levels?

Reply:
Much more I am afraid. We would have to parse PAGE inputs down to the requested hierarchy level, download the `AlternativeImage/@filename` from the workspace, process that, and reference the new file in another `AlternativeImage`, and finally serialise the PAGE again – all in bash.

Reply:
We could implement the cropping to specific regions in python and expose it as a CLI. It's still a lot of work to design, implement and test though.